packages/bridge/src/runtime/capi.legacy.ts

```diff
@@ -526,7 +527,7 @@ export const useFetch = (callback) => {
   onBeforeMount(() => !vm._hydrated && callFetches.call(vm))
 
   if (process.server || !isSsrHydration(vm)) {
-    if (process.client && !process.dev && process.static) { loadFullStatic(vm) }
+    if (process.client && isFullStatic) { loadFullStatic(vm) }
```

Comment:
I referred to the following code.
https://github.com/nuxt-community/composition-api/blob/main/src/runtime/composables/fetch.ts#L345